components/gitpod-db/src/typeorm/linked-in-profile-db-impl.ts

```diff
@@ -41,4 +41,9 @@ export class LinkedInProfileDBImpl implements LinkedInProfileDB {
             creationTime: existingProfile?.creationTime || new Date().toISOString(),
         });
     }
+
+    public async deleteProfile(userId: string): Promise<void> {
+        const repo = await this.getRepo();
+        await repo.delete({ userId });
```

Comment:
Will this throw if the userId does not exist?

Reply:
The documentation says:

> Deletes entities by a given criteria. Unlike save method executes a primitive operation without cascades, relations and other operations included. Executes fast and efficient DELETE query. Does not check if entity exist in the database.